registration/tests/test_views.py

```diff
@@ -592,7 +592,7 @@ def test_modify_user(self):
         application = BetaTestApplication.objects.get()
         self._assert_application_matches_form_data(application)
 
-    @override_settings(VARIABLES_NOTIFICATION_EMAIL='notifications@opencraft.com', INSTANCE_STORAGE_TYPE='s3')
+    @override_settings(VARIABLES_NOTIFICATION_EMAIL='notifications@opencraft.com')
```

Comment:
@lgp171188 Any reason for this change?

Reply:
@giovannicimolin, this was a change that I picked up from the `master` branch to fix a failing unit test as that hasn't yet been merged to the `new-console` branch, on which this PR branch is based.